services/crypto_add_live_hash.proto

```diff
@@ -31,45 +43,59 @@ import "basic_types.proto";
+ * A Live Hash value associating some item of content to an account.
+ *
+ * This message represents a desired entry in the ledger for a SHA-384
+ * hash of some content, an associated specific account, a list of authorized
+ * keys, and a duration the live hash is "valid".
  */
 message LiveHash {
     /**
-     * The account to which the livehash is attached
+     * An account associated via this live hash to the hashed content.
      */
     AccountID accountId = 1;
 
     /**
-     * The SHA-384 hash of a credential or certificate
+     * A SHA-384 hash of some content that is associated to the account
+     * or account holder.
      */
     bytes hash = 2;
 
     /**
-     * A list of keys (primitive or threshold), all of which must sign to attach the livehash to an account, and any one of which can later delete it.
+     * A list of keys, all of which MUST sign the transaction to add the
+     * live hash.<br/>
+     * Any one of these keys may, however, remove the live hash to revoke
+     * the association.
      */
     KeyList keys = 3;
 
     /**
-     * The duration for which the livehash will remain valid
+     * A duration describing how long this Live Hash SHALL remain valid.<br/>
+     * A Live Hash SHOULD NOT be relied upon after this duration has elapsed.
      */
     Duration duration = 5;
 }
 
 /**
- * At consensus, attaches the given livehash to the given account.  The hash can be deleted by the
- * key controlling the account, or by any of the keys associated to the livehash.  Hence livehashes
- * provide a revocation service for their implied credentials; for example, when an authority grants
- * a credential to the account, the account owner will cosign with the authority (or authorities) to
- * attach a hash of the credential to the account---hence proving the grant. If the credential is
- * revoked, then any of the authorities may delete it (or the account owner). In this way, the
- * livehash mechanism acts as a revocation service.  An account cannot have two identical livehashes
- * associated. To modify the list of keys in a livehash, the livehash should first be deleted, then
- * recreated with a new list of keys.
+ * Add a hash value to the ledger and associate it with an account.
+ *
+ * Create an entry in the ledger for a SHA-384 hash of some content, and
+ * associate that with a specific account.  This is sometimes used to associate
+ * a credential or certificate with an account as a public record.<br/>
+ * The entry created is also associated with a list of keys, all of which
+ * MUST sign this transaction.<br/>
+ * The account key for the associated account MUST sign this transaction.<br/>
+ *
+ * The live hash, once created, MAY be removed from the ledger with one
+ * or more signatures.
+ * - The account key of the account associated to the live hash.
+ * - Any one key from the key list in the live hash entry.
+ * - Any combination of keys from the key list in the live hash entry.
  */
```

Comment:
This transactionBody might be missing heading `### Block Stream Effects`.

Reply:
This entire transaction is deprecated, and has no block stream effects.